jekyll/_cci2/creating-orbs.md

```diff
@@ -495,7 +492,7 @@ This command is used to publish an orb. The following parameters may be passed w
 * attach-workspace: Boolean for whether or not to attach to an existing workspace. Default is false.
 * workspace-root: Workspace root path that is either an absolute path or a path relative to the working directory. Defaults to '.' (the working directory)
 
-### Validate and Publish Example
+### 11) Validate and Publish Example
```

Comment:
Prefix with Step

Reply:
This needs to be prefixed with `Step 11 - ` for consistency

Reply:
Removed the "11" from the step to remain consistent with rest of document